src/lang/evaluation.ml

```diff
@@ -290,27 +295,24 @@ and apply f l =
     | _ :: l -> pos l
     | [] -> f.Value.pos
   in
-  (* Position of the whole application. *)
   let pos = pos l in
-  let pos_f = f.Value.pos in
-  let mk ~pos v = { pos; Value.value = v } in
+  *)
+  (* NB: the above is more precise (we get the arguments), but I don't think
+     this is worth the price: we compute it for every application... *)
```

Comment:
I'd either remover it or make it optionally actionable. As comment, the code is bound to become obsolete..